conf/mesos-site.xml

```diff
@@ -49,7 +49,7 @@
 
   <property>
     <name>mesos.hdfs.cluster.name</name>
```

Comment:
Let's change this property to framework name, not cluster name. We are using this as the framework name here.